client/data/index.js

```diff
@@ -15,3 +15,4 @@ export * from './transactions';
 export * from './charges';
 export * from './timeline';
 export * from './disputes';
+export * from './settings';
```

Comment:
I was wondering why are exporting here all properties within settings (same question with the other exports). Won't we just need to export the hooks property?

Reply:
Good point! At the moment the settings data store is accessed exclusively through hooks so I will export only hooks here. This can be adjusted in the future if ever needed.